pyproject.toml

```diff
@@ -98,3 +98,18 @@ all = [
 [tool.isort]
 profile = "black"
 known_third_party = ["fastapi", "pydantic", "starlette"]
+
+[tool.pytest.ini_options]
+addopts = [
+  "--strict-config",
+  "--strict-markers",
+]
+xfail_strict = true
+junit_family = "xunit2"
+filterwarnings = [
+    "error",
+    'ignore:"@coroutine" decorator is deprecated since Python 3\.8, use "async def" instead:DeprecationWarning',
+    'ignore:The explicit passing of coroutine objects to asyncio\.wait\(\) is deprecated since Python 3\.8:DeprecationWarning',
+    'ignore:int_from_bytes is deprecated, use int\.from_bytes instead:cryptography.utils.CryptographyDeprecationWarning',
+    'ignore:Exception ignored in. <socket\.socket fd=-1:pytest.PytestUnraisableExceptionWarning',
```

Comment:
```suggestion
```

I wasn't able to trigger this warning. Maybe it was from Starlette <= `0.13.6` (I upgraded recently)?

Reply:
I think this might be only in the Proactor

Reply:
also PytestUnraisableExceptionWarning are susceptible to being flakey

Reply:
I believe `<socket\.socket fd=-1` comes from closing the ssl wrapped without closing the underlying socket